mempool/mempool.go

```diff
@@ -36,6 +37,9 @@ func (p *Pool) Enqueue(userTxn comettypes.Tx) error {
 	// NOTE: we should do reads and writes on the same view. Right now they occur on separate views.
 	// Unfortunately, comet's DB interface doesn't support it.
 	// Moving to a different DB interface is left for future work.
+	if txs, _ := rolluptypes.AdaptCosmosDepositTxToEthTx(userTxn); len(txs) > 0 {
```

Comment:
Right now, it appears like the comment applies to these lines. Let's leave a space in between, with a comment right above your changes explaining the reasoning (to disallow deposit txs in the mempool).